src/node/history.h

```diff
@@ -734,7 +734,7 @@ namespace ccf
       }
 
       const auto raw_cert = service_info->cert.raw();
-      const std::vector<const uint8_t> root_hash{
```

Comment:
libstdc++ doesn't like const vector?..

Reply:
No, there's a bizarre message about a non-const lvalue, I didn't look into it too closely because the scope where this is used is extremely limited. Also I think this should be refactored to use `std::span` anyway.